tests/sentry/api/serializers/test_group_stream.py

```diff
@@ -83,3 +84,33 @@ def test_profiling_issue(self):
         assert serialized["issueType"] == str(ProfileFileIOGroupType.slug)
         assert [stat[1] for stat in serialized["stats"]["24h"][:-1]] == [0] * 23
         assert serialized["stats"]["24h"][-1][1] == 1
+
+
+@region_silo_test
+class ExternalIssueSerializerTestCase(TestCase, APITestCase):
+    def test_external_issue_serializer(self):
+        group = self.create_group()
+        integration = self.create_integration(
+            organization=group.organization,
+            provider="jira",
+            external_id="some_id",
+            name="Jira",
+            metadata={"base_url": "https://example.com"},
+        )
+        external_issue = self.create_integration_external_issue(
+            group=group,
+            integration=integration,
+            key="APP-123",
+            title="this is an example title",
+            description="this is an example description",
+        )
+        req = self.make_request()
+        result = serialize(
+            [group],
+            request=req,
+            serializer=ExternalIssueSerializer(groups=[group]),
```

Comment:
the groups should not be passed to the constructor of the serializer -- that is something weird the previous serializer does that we do not need to do here